### PR TITLE
rotational-cipher: Added template to stub file

### PR DESCRIPTION
**Pull request by ZapAnton**
Contributes to #551

### CODE REVIEW

File: exercises/rotational-cipher/src/lib.rs [hunk 1]
```diff
@@ -1 +1,7 @@
-
+pub fn rotate(input: &str, key: u8) -> String {
```
Comment (coriolinus) [inline exercises/rotational-cipher/src/lib.rs]: None of the tests shift by negative keys, but there are plenty of people who would try shifting by `-1` before thinking of using `25`. I'd make `key` an `i8`, just as a hint to the students.

**Comment by ZapAnton** [comment]
Changed the `key` type. Should generalize the function and increase learning for the students

**Comment by ZapAnton** [comment]
Off-topic, if I may: Would it be practical to add some sort of web-hook or bot for this repo to schedule automatic PR merges and other tasks (like on the Rust language repo)?

**Comment by coriolinus** [comment]
There's been some discussion about integrating a github app to do that, but the available options haven't convinced us as yet. I'm wary of giving any external app write permission to the repo.

**Pull request merged**